File: CODES/1) C++/4) IF/14.cpp
#include <iostream>

using namespace std;

int main()
{
	int a, b, c, max, min;
	cout << " A = ";
	cin >> a;
	cout << " B = ";
	cin >> b;
	cout << " C = ";
	cin >> c;

	if (a > b and a > c)
	{
		max = a;
	}
	if (b > a and b > c)
	{
		max = b;
	}
	if (c > a and c > b)
	{
		max = c;
	}

	if (a < b and a < c)
	{
		min = a;
	}
	if (b < a and b < c)
	{
		min = b;
	}
	if (c < a and c < b)
	{
		min = c;
	}

	cout << " min : " << min << endl;
	cout << " max : " << max;
	return 0;
}

File: LANGUAGES/4) Php/4) IF/14.php
<?php
function katta($a, $b)
{
    if ($a > $b) return $a;
    return $b;
}
$a = 9;
$b = 7;
$c = 5;
echo " a = " . $a . " <br>";
echo " b = " . $b . " <br>";
echo " c = " . $c . " <br>";
if ($a > $b and $b < $c) {
    $min = $b;
    $max = katta($a, $c);
}
if ($a > $c and $c < $b) {
    $min = $c;
    $max = katta($a, $b);
}
if ($b > $a and $a < $c) {
    $min = $a;
    $max = katta($c, $b);
}
echo " min = " . $min . " max = " . $max . " <br>";
